### ISSUE TITLE
experience is atrocious. (0.15.11 or 0.16.2)

### ISSUE THREAD

**Issue by Anatharias** [jellyfin/jellyfin-androidtv#2952]
### Describe the bug

I got an LG TV on Web OS using the WebOS Jellyfin client and the experience is what it should be (8th gen i5 server with Intel transcoding). It is smooth, we can go fast forward, rewind, it's snappy. So this is my benchmark. Wife compliant!

Now, on my latest Fire TV 4K stick (which is really a fast little device - connected through Ethernet USB-Host) the Jellyfin player version 0.15.11 is a disaster, so I installed version 0.16.2 and it's just the same.
The app keeps on crashing. Cannot fast forward without a crash or if no crash, a real long wait before it resumes. cannot rewind, same behaviour.

If I don't "fore quit" the app from the FireTV settings, it will crash playback the next day (the two first crash logs are because I didn't force quit today).

On a minor annoyance, the menu item for audio sync is so big on screen that you cannot see the characters talking half of the time. The FireStick screen saver got over the playback twice as I'm writing this piece... and now, for a good 15 minutes, no screen saver... what gives

Anyway, this Android TV app is not wife compliant, if you see what I mean...

### Logs

All my logs back to back:
---
client: Jellyfin for Android TV
client_version: 0.16.0-beta.2
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
org.jellyfin.sdk.api.client.exception.TimeoutException: HTTP request timed out
	at org.jellyfin.sdk.api.client.KtorClient.request$suspendImpl(KtorClient.kt:117)
	at org.jellyfin.sdk.api.client.KtorClient$request$1.invokeSuspend(Unknown Source:19)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:103)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@dd607c3, Dispatchers.IO]
Caused by: io.ktor.client.features.HttpRequestTimeoutException: Request timeout has expired [url=http://192.168.2.201:8096/Users/c663babc-9dd5-4a5a-be6e-00818a6658b3/Items/f15427ff-4ec0-31f7-c492-71d16bbaaf2c, request_timeout=30000 ms]
	at io.ktor.client.features.HttpTimeout$Feature$install$1$1$killer$1.invokeSuspend(HttpTimeout.kt:149)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	... 4 more

```  
***Logcat***: 
```log
08-08 20:20:24.569 E/ItemRowAdapter(  597): 	at java.lang.reflect.Method.invoke(Native Method)
08-08 20:20:24.569 E/ItemRowAdapter(  597): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
08-08 20:20:24.569 E/ItemRowAdapter(  597): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
08-08 20:20:24.569 E/ItemRowAdapter(  597): Caused by: com.android.volley.TimeoutError
08-08 20:20:24.569 E/ItemRowAdapter(  597): 	at com.android.volley.toolbox.NetworkUtility.shouldRetryException(NetworkUtility.java:162)
08-08 20:20:24.569 E/ItemRowAdapter(  597): 	at com.android.volley.toolbox.BasicNetwork.performRequest(BasicNetwork.java:145)
08-08 20:20:24.569 E/ItemRowAdapter(  597): 	at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:132)
08-08 20:20:24.569 E/ItemRowAdapter(  597): 	at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:111)
08-08 20:20:24.569 E/ItemRowAdapter(  597): 	at com.android.volley.NetworkDispatcher.run(NetworkDispatcher.java:90)
08-08 20:20:26.125 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"d683de0cf2bd47de8eccd2e8cc4a37d9","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"97.45238095238095"}}
08-08 20:20:26.126 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"01260015c1744ae1b8a01d989e803aab","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"97.45238095238095"}}
08-08 20:20:29.057 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"82fefb17a74b4a6397692aeff9c64dd8","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"97.57142857142857"}}
08-08 20:20:29.057 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"54cbc11881794497909e8f6e922cdad0","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"97.57142857142857"}}
08-08 20:20:34.456 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"731acc514c8d4e638aa49fe090cf8efb","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"97.67261904761905"}}
08-08 20:20:34.456 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"39b76600a13e4b97abebd4a39a64af31","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"97.67261904761905"}}
08-08 20:20:35.790 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"f94b2ef97c63400aaada0faa1b3d8e59","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"97.79166666666666"}}
08-08 20:20:35.791 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"7e7abcf5bcde4f4abceb66dc344fc0ab","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"97.79166666666666"}}
08-08 20:20:37.147 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"cf736aa74cf041acaab6962e92d2daa6","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"97.88690476190476"}}
08-08 20:20:37.147 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"8adda71fc7df447a9d857bf1be949278","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"97.88690476190476"}}
08-08 20:20:39.817 E/jellyfin-apiclient(  597): VolleyError com.android.volley.TimeoutError: null
08-08 20:20:39.817 E/jellyfin-apiclient(  597): com.android.volley.TimeoutError
08-08 20:20:39.817 E/jellyfin-apiclient(  597): 	at com.android.volley.toolbox.NetworkUtility.shouldRetryException(NetworkUtility.java:162)
08-08 20:20:39.817 E/jellyfin-apiclient(  597): 	at com.android.volley.toolbox.BasicNetwork.performRequest(BasicNetwork.java:145)
08-08 20:20:39.817 E/jellyfin-apiclient(  597): 	at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:132)
08-08 20:20:39.817 E/jellyfin-apiclient(  597): 	at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:111)
08-08 20:20:39.817 E/jellyfin-apiclient(  597): 	at com.android.volley.NetworkDispatcher.run(NetworkDispatcher.java:90)
08-08 20:20:42.164 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"6498f6a3d5be4816aac0591525ac1b4b","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"97.98809523809524"}}
08-08 20:20:42.165 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"43a6ec7daafb4793a314b125656da943","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"97.98809523809524"}}
08-08 20:20:44.731 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
08-08 20:20:45.133 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"42183549a52943ad8346b5bb571b49d2"}
08-08 20:20:45.856 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"e9bfaecf1a704985bb8c2074ce992b6c","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.08928571428572"}}
08-08 20:20:45.856 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"e9daff6fa02246cab619040131dd7d89","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.08928571428572"}}
08-08 20:20:49.847 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"1163629bd120413c8530633d7c76ba9d","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.10714285714286"}}
08-08 20:20:49.848 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"676b70842355409c8e8c444f2092e31c","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.10714285714286"}}
08-08 20:20:53.026 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"8c706862598d44f6952dff8d64f3be67","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.11904761904762"}}
08-08 20:20:53.026 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"1c0ab937e4244ba8a5a23edcd352ec1f","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.11904761904762"}}
08-08 20:20:56.807 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"e530b37cf25b4532accf369049e89e2b","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.125"}}
08-08 20:20:56.807 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"dfc93dfbd46d4e669f325a72acd90a22","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.125"}}
08-08 20:20:59.686 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"ecf8c0b6600c4e1d88cb724160e9d067","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.14880952380952"}}
08-08 20:20:59.686 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"8a03c7028e7b4c248847bbdcdbf213d6","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.14880952380952"}}
08-08 20:21:03.112 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"d1531bc72b864668872767546678d9f7","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.16666666666666"}}
08-08 20:21:03.113 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"5e9af1125d0b4f42be9e8bc28a26164f","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.16666666666666"}}
08-08 20:21:05.205 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"335e2caaf3e744b586392bf6f1e73c4c","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.26190476190476"}}
08-08 20:21:05.205 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"422c7455c396499aa1a125d8a8ad1f83","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.26190476190476"}}
08-08 20:21:08.651 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"712c5db350ff48e0b1f8792f3faf0abc","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.36309523809524"}}
08-08 20:21:08.651 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"e4c7ce2cff3344d8a559f6e7da1ed22b","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.36309523809524"}}
08-08 20:21:08.841 I/HomeRowsFragment(  597): Updating audio queue in HomeFragment (onResume)
08-08 20:21:08.841 I/org.jellyfin.sdk.api.client.KtorClient(  597): GET http://192.168.2.201:8096/Users/c663babc-9dd5-4a5a-be6e-00818a6658b3/Items/f15427ff-4ec0-31f7-c492-71d16bbaaf2c
08-08 20:21:09.107 I/lyfin.androidt(  597): NativeAlloc concurrent copying GC freed 61184(2MB) AllocSpace objects, 0(0B) LOS objects, 39% free, 9MB/15MB, paused 133us total 111.586ms
08-08 20:21:10.248 W/RecyclerView(  597): No adapter attached; skipping layout
08-08 20:21:11.796 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"0266bd0375784c1daa5d7245a67361da","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.55952380952381"}}
08-08 20:21:11.796 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"77f77afde7734abf8aeb076e6b877c36","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.55952380952381"}}
08-08 20:21:12.986 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"82b5055a166c42ab9990bf543ada7afe","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.66071428571428"}}
08-08 20:21:12.986 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"e1efa6d37ad645af85adfe0c485ad93f","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.66071428571428"}}
08-08 20:21:13.100 I/HomeRowsFragment(  597): Updating audio queue in HomeFragment (onResume)
08-08 20:21:13.100 I/org.jellyfin.sdk.api.client.KtorClient(  597): GET http://192.168.2.201:8096/Users/c663babc-9dd5-4a5a-be6e-00818a6658b3/Items/f15427ff-4ec0-31f7-c492-71d16bbaaf2c
08-08 20:21:13.381 I/lyfin.androidt(  597): NativeAlloc concurrent copying GC freed 51472(2MB) AllocSpace objects, 0(0B) LOS objects, 39% free, 9MB/15MB, paused 136us total 122.620ms
08-08 20:21:14.733 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
08-08 20:21:14.739 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"c80ae814f3184e02b19faf2bbcd4ed08"}
08-08 20:21:16.803 I/lyfin.androidt(  597): NativeAlloc concurrent copying GC freed 39758(1785KB) AllocSpace objects, 0(0B) LOS objects, 40% free, 8MB/14MB, paused 139us total 106.191ms
08-08 20:21:17.800 I/org.jellyfin.sdk.api.client.KtorClient(  597): GET http://192.168.2.201:8096/DisplayPreferences/d565273fd114d77bdf349a2896867069?userId=c663babc-9dd5-4a5a-be6e-00818a6658b3&client=jellyfin-androidtv
08-08 20:21:20.201 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"6e048ab024d54cf1aa5a81b1394c22c7","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.76190476190476"}}
08-08 20:21:20.201 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"136b00e9f8c8438caad0da41c027bd99","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.76190476190476"}}
08-08 20:21:30.190 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"2bb090f11c39493e804b312e9b0d9616","Data":{"ItemId":"429cfa5c1648e89c5f696f08e19b138f","Progress":"98.85714285714286"}}
08-08 20:21:30.190 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(  597): Receiving (raw) message {"MessageType":"RefreshProgress","MessageId":"16d724dd1d1b448a8d270388831e008f","Data":{"ItemId":"db4c1708cbb5dd1676284a40f2950aba","Progress":"98.85714285714286"}}
08-08 20:21:38.860 E/ACRA    (  597): ACRA caught a TimeoutException for org.jellyfin.androidtv
08-08 20:21:38.860 E/ACRA    (  597): org.jellyfin.sdk.api.client.exception.TimeoutException: HTTP request timed out
08-08 20:21:38.860 E/ACRA    (  597): 	at org.jellyfin.sdk.api.client.KtorClient.request$suspendImpl(KtorClient.kt:117)
08-08 20:21:38.860 E/ACRA    (  597): 	at org.jellyfin.sdk.api.client.KtorClient$request$1.invokeSuspend(Unknown Source:19)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun.resumeRootWith(SuspendFunctionGun.kt:194)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun.access$resumeRootWith(SuspendFunctionGun.kt:15)
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.util.pipeline.SuspendFunctionGun$continuation$1.resumeWith(SuspendFunctionGun.kt:89)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:103)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
08-08 20:21:38.860 E/ACRA    (  597): 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@dd607c3, Dispatchers.IO]
08-08 20:21:38.860 E/ACRA    (  597): Caused by: io.ktor.client.features.HttpRequestTimeoutException: Request timeout has expired [url=http://192.168.2.201:8096/Users/c663babc-9dd5-4a5a-be6e-00818a6658b3/Items/f15427ff-4ec0-31f7-c492-71d16bbaaf2c, request_timeout=30000 ms]
08-08 20:21:38.860 E/ACRA    (  597): 	at io.ktor.client.features.HttpTimeout$Feature$install$1$1$killer$1.invokeSuspend(HttpTimeout.kt:149)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
08-08 20:21:38.860 E/ACRA    (  597): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
08-08 20:21:38.860 E/ACRA    (  597): 	... 4 more

```  

### App information

***App version***: `0.16.0-beta.2` (`160002`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"kara","BOOTLOADER":"unknown","BRAND":"Amazon","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"kara","DISPLAY":"PS7646.3554N","FINGERPRINT":"Amazon\/kara\/kara:9\/PS7646.3554N\/0028085969408:user\/amz-p,release-keys","HARDWARE":"mt8696","HOST":"i3-ri-14-use1a-b-16-04-fos-102","ID":"PS7646.3554N","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"Amazon","MODEL":"AFTKA","PERMISSIONS_REVIEW_REQUIRED":false,"PRODUCT":"kara","RADIO":"unknown","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"amz-p,release-keys","TIME":1685919945000,"TYPE":"user","UNKNOWN":"unknown","USER":"build","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"PPR1.180610.009","CODENAME":"REL","FIRST_SDK_INT":28,"INCREMENTAL":"0028085969540","PREVIEW_SDK_INT":0,"RELEASE":"9","RESOURCES_SDK_INT":28,"SDK":"28","SDK_INT":28,"SECURITY_PATCH":"2022-01-01"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":160002,"VERSION_NAME":"0.16.0-beta.2"}
```  

### Device information

***Android version***: `9`  
***Device brand***: `Amazon`  
***Device product***: `kara`  
***Device model***: `AFTKA`  

### Crash information

***Start time***: `2023-08-08T20:19:41.875-04:00`  
***Crash time***: `2023-08-08T20:21:38.913-04:00`  

--------------------------------------------------------------------------------
--------------------------------------------------------------------------------

remainder of logs are here (too many chars):
https://rentry.co/trbcwm

### Application version

0.16.2

### Where did you install the app from?

None

### Device information

Amazon FireStick 4k latest gen

### Android version

Fire OS 7.3.2.8

### Jellyfin server version

10.8.10

**Comment by nielsvanvelzen** [comment]
I have a lot of things to say but I'll just give you this link and perhaps you'll understand why I closed this issue.

https://jellyfin.org/docs/general/community-standards#code-of-conduct

**Comment by Anatharias** [comment]
English is not my first language, the strength of words might not resonate the same way for you as it does for me.

Even though you feel that my bug report violates the code of conduct, I have **not** stated that "one has made a crap app", nor did antagonize, flamed, insulted, etc., anyone.
What I described was my atrocious experience of the app, based on another device to compare it to.
Maybe this is a problem with the fire stick, maybe a problem with my server. Regardless, I described it bluntly, sure, but politely and realistically.

So if politely describing an atrocious experience is against the rules, so be it.
